src/foam/u2/Element.js

```diff
@@ -528,90 +528,140 @@ foam.CLASS({
     'onunload'
   ],
 
-  constants: {
-    // Psedo-attributes don't work consistently with setAttribute()
-    // so need to be set on the real DOM element directly.
-    PSEDO_ATTRIBUTES: {
-      value: true,
-      checked: true
+  constants: [
+    {
+      documentation: `
+        Psedo-attributes don't work consistently with setAttribute() so need to
+        be set on the real DOM element directly.
+      `,
+      name: 'PSEDO_ATTRIBUTES',
+      value: {
+        value: true,
+        checked: true
+      },
     },
 
-    DEFAULT_VALIDATOR: foam.u2.DefaultValidator.create(),
+    {
```

Comment:
Is this so it can be serialized?

Reply:
Yeah otherwise it gets serialized as:
DEFAULT_VALIDATOR: { class: 'foam.u2.DefaultValidator' }
And with that, there's no way of knowing for sure whether this should be converted to an FObject or left as a regular Object so making it use a factory makes it clear what the value should be (and it can be lazier this way).